ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -219,8 +217,6 @@ const Footer = () => {
     } else {
       dispatch(resolvePendingApproval(currentConfirmation.id, undefined));
     }
-    dispatch(updateCustomNonce(''));
-    dispatch(setNextNonce(''));
```

Comment:
These calls were added here as it at times resulted in confirmation using old nonce when switching networks. Can you plz check if removing this does not re-introduce that bug.

Reply:
After discussing this with @jpuri, she informed me that there used to be a bug in the legacy confirmations that:
* if you created confirmation on a network
* edit the nonce
* and then switch to another network
* the confirmation page now would show old nonce.

Thus we added this code to clear the old nonce value after confirmation was submitted.

I tested this scenario a few times and it seems to not be an issue in the new confirmations even when removing this code. I will ask our QA engineer to double check this for us.